package.json

```diff
@@ -147,7 +150,11 @@
     "typedoc": "^0.27.6",
     "typescript": "~5.2.0"
   },
-  "packageManager": "yarn@3.8.1",
+  "packageManager": "yarn@4.6.0",
+  "volta": {
+    "node": "22.14.0",
+    "yarn": "4.6.0"
+  },
```

Comment:
What's the benefit of Volta?

Reply:
Interjects calls to the tools and links them to the specified version. 
The result is that devs can switch between projects / branches and always have the correct version. 

If a dev has not the Volta shim installed this is noop, but I would recommend it as that way it is easier in case of behaviour differences between versions.